platform/core/flatbuffers_shared/setup_flatbuffers.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -e    # Exit if any command fails
+
+echo '===> Setting up flatbuffers'
+
+FLATBUFFERS_VERSION=v2.0.0
```

Comment:
would it make sense to have this as an env setting outside of the script so it can be reused for the cargo build too?

Reply:
currently this is used only in this script. Cargo will not specify the version in Cargo.toml, the version is controlled with `--branch $FLATBUFFERS_VERSION` in `git clone --branch $FLATBUFFERS_VERSION $FLATBUFFERS_REPO`, and Cargo will just link to the Rust crate.

Reply:
Tend to agree with @irstavr . Configuration is always best in the environment, and Makefiles do have a place for that. Also, you can change the values of Makefile environment variables in the invocation, should it be needed.

Reply:
Sure, but this is meant to be used only in this file.